### ISSUE TITLE
Typescript: CollectionQuery doesn't allow returning null from query function

### ISSUE THREAD

**Issue by damonmaria** [IjzerenHein/firestorter#57]
[Release 0.16.0](https://github.com/IjzerenHein/firestorter/releases/tag/v0.16.0) added the ability to return `null` from query functions but `null` isn't a valid return type in the Typescript declaration:
```typescript
export type CollectionQuery =
	| ((ref: firestore.CollectionReference) => firestore.Query | undefined)
	| firestore.Query;
```

**Comment by IjzerenHein** [comment]
Hey thanks for letting me know. I've fixed the problem on master and will push a new release within the next couple of days.
Cheers

**Comment by IjzerenHein** [comment]
Hi, it's taken way too long, but I have finally released a new version with the fix for this, v1.2.0.
Cheers and let me know whether I can close this issue,
Hein
